### DISCUSSION TITLE
Difficulty getting recognizable face with Dreambooth in updated colab

### DISCUSSION THREAD

**Discussion by thegoodtroll** [TheLastBen/fast-stable-diffusion#1251] [Q&A] [answered]
I am experiencing difficulty training recognizable faces ever since the updated colab. I used to get REALLY good results (dating back to 27th december 2022) training on 24 images for approx. 1300 steps for UNet and 400 steps for text_encoder with the old learning rates (believe they were 2e-6 for both before). Now, with the same settings, it doesn't even recognize my class. I have found that if I change the learning rate for the UNet to 5e-6 and train for 500-1000 steps it begins to recognize the class. Haven't experienced with the learning rates for the text_encoder yet.

Does anyone have any insights to the changed colab/ models and its new settings, or maybe ideas on how to achieve the previous high quality classes in terms of recognizability? I don't really possess any theoretical insights to these models, so all my experience with the hyperparameters is purely empirical.

Otherwise, thanks TheLastBen for an AWESOME colab, which I have had great use of these past few months.

**Comment by TheLastBen** [comment, marked as answer]
for the 2e-6 learning rate, you need to go as high as 5000 steps to get good results. For the default 1e-5, you need only 800 steps for 10 instance images.
Also, try the V2-512, it's a good model

**Reply by thegoodtroll** [reply]
I'll definitely try the V2.1-512 model, thanks. When I run with the default learning rates in the notebook for 1000 steps UNet and 300 steps for text_encoder it cant even recognize the object. But when I switch to the 5e-6 learning rate for the UNet it begins to recognize way better for the same number of steps... for the 768 model that is

**Reply by TheLastBen** [reply]
the default learning rate is way higher than 5e-6, it's 1e-5, try it

**Reply by thegoodtroll** [reply]
I mean, I do use the default learning rates as they are now for both the text_encoder as well as the UNet. But they yielded terrible results when running for 300 steps and 800 steps respectively... When I change the learning rate for the UNet to 5e-6 and run for 1000 steps it seems to produce better results

**Reply by thegoodtroll** [reply]
Interesting... just tried it with default settings for the 512 V2.1 model and it produces really good results...

**Reply by TheLastBen** [reply]
the V2-512 and 768 handle high lr very well
